### PR TITLE
Query Multiple Tables using Join

### PR DESCRIPTION
**Pull request by Gambit142**
I did the following in the join-branch:

- Created two tables 
  - _**Owners Table**_
  - _**Species table**_

- Added foreign keys to the _**animals table**_ and created two new columns in the table
  - _**species_id**_
  - _**owner_id**_

- Inserted values into the _**owners and species tables**_
- Modified the data in the _**animals table**_
- Queried multiple tables using `Join` method and `aggregate` functions.

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
-INSERT INTO animals (name, date_of_birth, escape_attempt, neutered, WEIGHT_KG) 
-VALUES ('Devimon', '2017-05-12', 5, true, 11);
-INSERT INTO animals (name, date_of_birth, escape_attempt, neutered, WEIGHT_KG) 
-VALUES ('Charmander', '2020-02-08', 0, false, -11), ('Plantmon', '2022-11-15', 2, true, -5.7),
-('Squirtle', '1993-04-02', 3, false, -12.13), ('Angemon', '2005-06-12', 1, true, -45),
- ('Boarmon', '2005-06-07', 7, true, 20.4),
-('Blossom', '1998-10-13', 3, true, 17);
\ No newline at end of file
+INSERT INTO species (name) VALUES ('Pokemon'), ('Digimon');
+
```
Comment (oxenprogrammer) [inline data.sql:8]: **Changes Required**
We are missing the following `UPDATES`
- [ ] if an animal's name ends with mon it should have Digimon as a species_id, all other animals should have Pokemon instead.
- [ ] all animals got assigned owners.

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi @Gambit142, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (Gambit142) [reply data.sql:8]: Hi @oxenprogrammer, thank you for the feedback. All animals have owners and I modified the animal's data to include the species of the animals. Kindly check the `data.sql` file for that. The project requirements say that we should modify the inserted data and not use updates. Kindly see the requirement in the snapshot below:

image

#### Kindly advise me on what to do. 🙏

**Review by descholar-ceo — APPROVED** [review]
Hi @Gambit142 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Gambit142** [comment]
Thank you

**Pull request merged**